flink-table/flink-sql-gateway-common/src/main/java/org/apache/flink/table/gateway/common/utils/SqlGatewayException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.gateway.common.utils;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+/** General exception for SQL gateway related errors. */
```

Comment:
```suggestion
/**
 * General Exception for all errors during SQL gateway serving.
 *
 * <p>This exception indicates that an internal error occurred or that a feature is not supported
 * yet. Usually, this exception does not indicate a fault of the user.
 */
```

Reply:
We'd better to make the assumption here. It's possible the wrong usage will get this error.